recipe/meta.yaml

```diff
@@ -47,6 +52,7 @@ requirements:
     - importlib-metadata
     - pyyaml >=3.13
     - packaging >=19.0
+    - scipy >=1.8
```

Comment:
One thing here: it's important to _never_ install "old" `astropy` and the new `astropy-based` in the same environment. This is easiest on a version boundary, rather than just a build number bump, as it could be:

```yaml
requirements:
  # ...
  run_constrained:
    - astropy >=6.1.4
```

As opposed to `astropy >=6.1.3_*_3` (or whatever build number this actually ships on).

Reply:
this looks good: can be resolved